sdl-core/src/main/scala/io/smartdatalake/workflow/action/executionMode/FileIncrementalMoveMode.scala

```diff
@@ -41,12 +41,20 @@ import org.apache.spark.sql.Row
  * A better implementation would be to observe files by a custom metric. Unfortunately there is a problem in Spark with that, see also [[CollectSetDeterministic]]
  * - Partition values preserved.
  *
- * @param archivePath if an archive directory is configured, files are moved into that directory instead of deleted, preserving partition layout.
+ * @param archivePath if an archive directory is configured, files are moved into that directory instead of deleted, preserving partition layout. See also `àrchiveInsidePartition` option.
```

Comment:
```suggestion
 * @param archivePath if an archive directory is configured, files are moved into that directory instead of deleted, preserving partition layout. See also `archiveInsidePartition` option.
```